docs/reference/tab-widgets/troubleshooting/data/restore-from-snapshot.asciidoc

```diff
@@ -0,0 +1,211 @@
+// tag::cloud[]
+In order to restore the indices and data streams that are missing data:
+
+**Use {kib}**
+
+//tag::kibana-api-ex[]
+. Log in to the {ess-console}[{ecloud} console].
++
+
+. On the **Elasticsearch Service** panel, click the name of your deployment.
++
+
+NOTE: If the name of your deployment is disabled your {kib} instances might be
+unhealthy, in which case please contact https://support.elastic.co[Elastic Support].
+If your deployment doesn't include {kib}, all you need to do is
+{cloud}/ec-access-kibana.html[enable it first].
++
+. Open your deployment's side navigation menu (placed under the Elastic logo in the upper left corner)
+and go to **Stack Management > Index Management**.
++
+. In this step we are going to find all the indices and data streams missing data. If you already have that information
+you could skip this. Go in the **Indices** tab, enable **Include hidden indices** toggle and find all the open indices
+that have an empty **Health** and empty **Data stream**, keep their names somewhere handy. Then, go to the
+**Data Streams** tab and keep the name of all the data streams whose **Health** is `red`.
++
+[role="screenshot"]
+image::images/troubleshooting/data/kibana-index-management-select-red-indices.png[{kib} indices,align="center"]
++
+[role="screenshot"]
+image::images/troubleshooting/data/kibana-index-management-select-red-data-steams.png[{kib} data streams,align="center"]
++
+. Use the left side menu to go to **Snapshot and Restore**, there you can verify that you have a snapshot to restore
+the target indices and data streams from. By clicking on every snapshot you can see the indices and data streams it
+contains. Not all resources need to be restored from the same snapshot. **Do not proceed** there is no workable
+backup of the affected resources in a snapshot, this will result in further data loss.
++
+[role="screenshot"]
+image::images/troubleshooting/data/kibana-restore-snapshot-content.png[{kib} snapshots,align="center"]
++
+. Go back to **Index Management** via the left side menu. In **Indices** enable **Include hidden indices** toggle and
+delete all the indices that have noted, by selecting them and then using the **Manage index** button
+choose delete index. In the **Data Streams** tab repeat the same by selecting the `red` data streams and then
```

Comment:
Would closing the index be enough? It seems that we're recommending closing them in the on-prem version

Reply:
Well it would be enough but the user experience is not as good. 

Let me elaborate, when a user restores a snapshot via the UI they cannot select a backing index, they can only select a data stream. This would mean that we have to explain to the user, that they need to go to the index, select all the backing indices of their data stream, close them, and then restore the data stream. I thought since we are trying to avoid confusing the user that we might want to stay away from backing indices etc. 

What do you think?

Reply:
Ah I see, thanks for the details